src/open_inwoner/pdc/views.py

```diff
@@ -225,6 +226,16 @@ def get_context_data(self, **kwargs):
         context["display_social"] = config.display_social
         return context
 
+    def render_to_response(self, context, **response_kwargs):
+        """
+        Override to post-process rendered HTML content
+        """
+        response = super().render_to_response(context, **response_kwargs)
+
+        response.add_post_render_callback(convert_styling_to_nlds)
```

Comment:
This modifies the whole document and not just the content from the field, so I don't think this is what we want.